src/main/java/picard/analysis/directed/CollectTargetedMetrics.java

```diff
@@ -98,6 +102,11 @@ protected abstract COLLECTOR makeCollector(final Set<MetricAccumulationLevel> ac
     @Option(doc="Sample Size used for Theoretical Het Sensitivity sampling. Default is 10000.", optional = true)
     public int SAMPLE_SIZE=10000;
 
+    @Option(doc="Output for Theoretical Sensitivity metrics.  Default is null.", optional = true)
```

Comment:
specify that this is for the values specified in  the `ALLELE_FRACTION` argument